packages/scene-composer/src/components/StateManager.tsx

```diff
@@ -157,7 +167,16 @@ const StateManager: React.FC<SceneComposerInternalProps> = ({
           },
         ];
       }
-
+      // Add entityID info part of additional component data
+      // We assumed IDataBindingMap will have only one mapping as data binding
+      // will always have only one entity data.
+      if (entityBindingComponent) {
+        additionalComponentData?.push({
```

Comment:
so if it's an empty node with just the entity binding component, we will not return entity binding data? I don't think this is expected behavior, and we should always return entity binding data when the selected node has this component